geomstats/geometry/lie_algebra.py

```diff
@@ -104,7 +104,7 @@ def basis_representation(self, matrix_representation):
         matrix_representation: array-like, shape=[n_sample, n, n]
 
         Returns
-        -------
+        ------
```

Comment:
I think the official numpy docstring guide (https://numpydoc.readthedocs.io/en/latest/format.html) also has 7 dashes below the Returns (instead of the 6 you have here)